### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by omarbabou**
**This pull request contains the following elements**:

      - Add HMTL-CSS/JS linters configuration
      - Create a Javascript file
      - Add the responsive menu and its design
      - Fix Linters errors.

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* eslint-disable comma-dangle */
+/* eslint-disable function-paren-newline */
+/* eslint-disable implicit-arrow-linebreak */
```
Comment (zieeco) [inline js/app.js]: - [x] **[REQUIRED CHANGES]**

- Please, let's try to write clean codes which conform to standards to make us better developers. try removing these lines and fix the linter error as this is not really a good reason to disable the eslint linter.

File: index.html [hunk 2]
```diff
@@ -25,15 +25,17 @@
...
-
       <nav>
-        <img src="./img/menu.svg" class="img" alt="" />
-        <div class="lists">
+        <a href="./index.html" id="logo">Idrissa</a>
+        <div class="icons">
+          <i class="fa fa-bars display-icon"></i>
+          <i class="fa fa-times no-display-icon"></i>
+        </div>
+        <div class="ul-list">
```
Comment (zieeco) [inline index.html:34]: - [x] **[REQUIRED CHANGES]**

- Please, try adjusting your styles so the menu container talked the whole device height. As of right now, a user can choose to ignore your menu list items and just scroll down to your page. Try to adjust to styles to improve the user experience. check out the video below to see your mobile menu in action:

https://user-images.githubusercontent.com/59397159/170011897-08e5f9e6-bdde-4849-842f-cfa95921b25f.mp4

Comment (zieeco) [reply index.html:34]: **[REQUIRED CHANGES]**

- Please, try reducing the size of your canceled image to match the Figma design

**you design**
canYours

**figma design**
canFigma

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @omarbabou 


Great work on your mobile menu 👏 
You've done well implementing the design and functionality, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
# Hi @omarbabou , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- _**(optional)**_ Nice job but I recommend blocking scroll ability when mobile menu is opened

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — COMMENTED** [review]
### Hi @omarbabou, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] What about preventing the sub-background from scrolling when the navbar is open? You can use a CSS property `over-flow: hidden;` to do that, which will make your mobile menu correspond to the original design.💯 🥇 

 - [ OPTIONAL] Nice implementing the design, however, it would be better if you add a little bit of blur to your overlay in order to match the Figma design and also make a good contrast 🎨 👨🏻‍🎨 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**